queries/nim/highlights.scm

```diff
@@ -0,0 +1,689 @@
+; SPDX-FileCopyrightText: 2023 Leorize <leorize+oss@disroot.org>, aMOPel <>
```

Comment:
Do you insist on this copyright statement? We'd prefer all queries to be available under the same license.

Reply:
@alaviss that is your copyright, right?

Reply:
I'd prefer having the MPL license kept for easier reintegration back into upstream.

Since @aMOPel wrote a lot of extra code for this, they can dual-license this file with `Apache-2.0` if they wish to do so.

Reply:
How would I do that?

Reply:
Add `; SPDX-License-Identifier: Apache-2.0` to the file

Reply:
done